server/command.go

```diff
@@ -278,6 +283,55 @@ func executeHelp(p *Plugin, c *plugin.Context, header *model.CommandArgs, args .
 	return responsef(helpText)
 }
 
+func executeImport(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	posts, err := p.API.GetPostsForChannel(header.ChannelId, 0, 1)
+	if err != nil {
+		return responsef(err.Error())
+	}
+
```

Comment:
2/5 It is my understanding that `GetPostsForChannel` may return a `nil` error, and yet an empty result set. We should check for it, and return a meaningful error message, explaining the usage (last post should have the attachment to import)